src/services/game-logic/pieces/Pawn.ts

```diff
@@ -76,6 +84,29 @@ class Pawn extends Piece {
             pieceMoves.push(m);
         });
     }
+
+    private enPassant(board: Board, pieceMoves: Square[]): void {
+        if (
+            (this.color === Colors.WHITE && this.position.row === 3) ||
+            (this.color === Colors.BLACK && this.position.row === 4)
+        ) {
+            const squareLeft = new Square(this.position.row, this.position.column - 1);
+            const squareRight = new Square(this.position.row, this.position.column + 1);
+
+            [squareLeft, squareRight].forEach((square) => {
```

Comment:
Niestety to dalej za mało. Sprawdzasz czy ostatni ruch był wykonany jakimkolwiek pionkiem o dwa pola i jeśli tak to dajesz możliwość bicia w przelocie obydwu pionków. Prowadzi to do błędu ze screena poniżej (ruch czarnego piona). 
![image](https://user-images.githubusercontent.com/41524451/107717621-5b909280-6cd4-11eb-9898-db65ebe19223.png)

Reply:
done